explore/launch/explore.launch

```diff
@@ -1,15 +1,16 @@
 <launch>
 <node pkg="explore_lite" type="explore" respawn="false" name="explore" output="screen">
-  <param name="robot_base_frame" value="base_link"/>
+  <param name="robot_base_frame" value="create1_tf/base_footprint"/>
+  <param name="move_base_topic" value="create1/move_base"/>
   <param name="costmap_topic" value="map"/>
   <param name="costmap_updates_topic" value="map_updates"/>
   <param name="visualize" value="true"/>
   <param name="planner_frequency" value="0.33"/>
   <param name="progress_timeout" value="30.0"/>
   <param name="potential_scale" value="3.0"/>
   <param name="orientation_scale" value="0.0"/>
-  <param name="gain_scale" value="1.0"/>
-  <param name="transform_tolerance" value="0.3"/>
+  <param name="gain_scale" value="0."/>
```

Comment:
What are these changes about?

Reply:
I removed these changes and will call them from create_autonomy, but summarizing:
`potential_scale` gives weigh to the nearer frontiers while
`gain_scale` gives weigh to the bigger frontiers.
Doing `gain_scale = 0` the robot will try to continue exploring always against the closest frontier, I realized that it's the best way.

Reply:
Put this explanation in `create_autonomy` PR, please.